docs/en/entity-onboarding.rst

```diff
@@ -0,0 +1,1095 @@
+- **Key Rotation**: Coordinate Federation Entity Key rotation with certificate renewal procedures.
+- **CRL Management**: For Protocol certificates with validity > 24 hours, maintain current CRL publication.
+- **Federation Exit**: Ensure proper certificate revocation during voluntary or supervisory body-initiated federation exit.
+
+
+Entity Lifecycle Management
+----------------------------
+
+This section provides technical implementation procedures for entity lifecycle management. For high-level lifecycle concepts and business processes, see :ref:`onboarding-high-level:Entity Lifecycle Management`.
+
+While administrative data update MUST follow governance processes that are out of scope of this technical specification, technical configuration updates are described in the following sections.
+
+Technical Configuration Updates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Technical updates affecting federation protocol operations MUST follow specific procedures for:
+
+  - **Certificate Renewal**
+
+    1. **Pre-renewal Preparation**: The Entity MUST generate a new CSR with updated certificate information.
+    2. **Renewal Request**: The Entity MUST submit renewal request with new CSR following the same technical procedure as initial onboarding.
+    3. **Certificate Integration**: The Entity MUST update its Entity Configuration with new certificate chain in ``x5c`` parameter.
+    4. **Trust Chain Validation**: The Entity MUST verify updated Trust Chain through ``/resolve`` endpoint.
+    5. **Registry Update**: The Entity SHOULD confirm updated entity information in Trust Anchor ``/list`` endpoint.
+
+  - **Key Rotation**
+
+    1. **New Key Generation**: The Entity MUST generate a new Federation Entity Public Key pair.
+    2. **Parallel Key Publication**: The Entity MUST publish both old and new keys in Entity Configuration ``jwks`` claim during transition period.
+    3. **Certificate Request**: The Entity MUST request new certificate for new public key following standard procedure.
+    4. **Gradual Migration**: The Entity MUST update Entity Configuration to use new key for signing while maintaining old key for verification.
+    5. **Old Key Deprecation**: The Entity MUST remove old key from Entity Configuration after validation period.
+
+  - **Metadata Updates**
+
+    - **Endpoint Changes**: The Entity MAY update service endpoints in entity-specific metadata.
+    - **Capability Updates**: The Entity MAY modify supported protocols, algorithms, or service capabilities within the constraints defined by this IT-Wallet implementation profile.
+
+
+
+All technical updates MUST be validated through:
+
+  1. **Entity Configuration Validation**: The Entity MUST verify updated EC structure and content.
+  2. **Trust Chain Resolution**: The Entity MUST confirm ``/resolve`` endpoint returns valid Trust Chain.
```

Comment:
iot seems that wallet providers are not mandated to implement the resolve endpoint

@grausof am I correct?